### PR TITLE
Openstack queens update

### PR DESCRIPTION
**Pull request by apdibbo**
Update the cloud web interface to use work with the Queens release of OpenStack

### CODE REVIEW

File: controllers/api/openstack/vm.py [hunk 1]
```diff
@@ -50,11 +51,11 @@ def PUT(self):
             # Aquilon
             meta = {}
 
-            if (json['archetype'] is not None) and (json['archetype'] != ''):
+            if ('archetype' in json):
```
Comment (tofu-rocketry) [inline controllers/api/openstack/vm.py]: The brackets are unnecessary.

File: controllers/api/openstack/vm.py [hunk 2]
```diff
@@ -50,11 +51,11 @@ def PUT(self):
...
                 meta['AQ_ARCHETYPE'] = json['archetype']
                 meta['AQ_PERSONALITY'] = json['personality']
 
-            if (json['sandbox'] is not None) and (json['sandbox'] != ''):
+            if ('sandbox' in json):
```
Comment (tofu-rocketry) [inline controllers/api/openstack/vm.py]: Unnecessary brackets.

File: controllers/api/openstack/machines.py [hunk 3]
```diff
@@ -17,6 +17,13 @@ class Machines(object):
     @cherrypy.tools.isAuthorised(redirect=True)
     @cherrypy.tools.jinja(template="machines/index.html")
     def index(self):
+
+        try:
+            username = cherrypy.request.cookie.get('fedid').value
+        except:
```
Comment (tofu-rocketry) [inline controllers/api/openstack/machines.py]: Shouln't use bare except. What exception do you get if you can't fetch a cookie?

File: controllers/api/openstack/templatelist.py [hunk 4]
```diff
@@ -18,18 +19,21 @@ def GET(self):
         auth_string = cherrypy.request.cookie.get('session').value
 
         novaClient = getNovaInstance()
```
Comment (tofu-rocketry) [inline controllers/api/openstack/templatelist.py:21]: Is nova still used somewhere or has it been completely replaced by glance?

**Review by tofu-rocketry — COMMENTED** [review]
Still a couple of outstanding comments.

File: controllers/api/openstack/templatelist.py (hunk 4 above)
Comment (apdibbo) [reply controllers/api/openstack/templatelist.py:21]: Yes, the nova is still the primary api but it no longer proxies requests to glance.

**Comment by apdibbo** [comment]
That should be them all addressed now

**Review by tofu-rocketry — APPROVED** [review]
Looks okay.

**Pull request merged**